modules/dxp/apps/commerce/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/diagram_type/svg.jsp

```diff
@@ -21,8 +21,25 @@ CSDiagramSettingDisplayContext csDiagramSettingDisplayContext = (CSDiagramSettin
 
 CPDefinition cpDefinition = csDiagramSettingDisplayContext.getCPDefinition();
 CSDiagramSetting csDiagramSetting = csDiagramSettingDisplayContext.fetchCSDiagramSetting();
-
-String type = SVGCSDiagramType.KEY;
 %>
 
-<%@ include file="/diagram_type/diagram.jspf" %>
\ No newline at end of file
+<div>
+	<span aria-hidden="true" class="loading-animation"></span>
+
+	<react:component
+		module="js/diagram/DiagramWithAutomapping"
+		props='<%=
+			HashMapBuilder.<String, Object>put(
+				"diagramId", csDiagramSetting.getCSDiagramSettingId()
+			).put(
+				"imageURL", csDiagramSettingDisplayContext.getImageURL()
+			).put(
+				"isAdmin", true
+			).put(
+				"pinsCSSSelector", "#Livello_Testi > text, [id*=MTEXT] > text"
```

Comment:
Isn't this a rather specific selector to set by default?

Reply:
Yes, that will be replace in the next PR by: 
```
).put(
	"pinsCSSSelectors", csDiagramSettingDisplayContext.getImageCSSSelectors()
```

I couldn't do it before this [PR](https://github.com/liferay-commerce/liferay-portal/pull/1764) got merged but I needed to see it working in portal